### PR TITLE
Include FAQ to README

### PR DESCRIPTION
**Pull request by Jibola**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -101,6 +101,29 @@ from the new application's `apps.py` file.
...
+- Able to define MongoDB collections using Django models
+- Able to query data using Django’s `QuerySets`
+- Able to use Django’s authentication system `django.contrib.auth`
+- Able to use Django’s automatic admin interface `django.contrib.admin`
+- Able to use Django’s management commands
+- Able to use MongoDB’s Aggregation Pipeline
+
+## What's next?
+We are working towards a Public Preview release early 2025. We do not yet have a timeline for general availability. Exact details of the roadmap will be published soon. Stay tuned! 
+What level of support will MongoDB provide to developers using this code?
```
Comment (blink1073) [inline README.md:123]: Is this meant to be a new header?

File: README.md [hunk 2]
```diff
@@ -101,6 +101,29 @@ from the new application's `apps.py` file.
...
+
+According to the 2023 Django Developer Survey, 8% of respondents reported using MongoDB as a database backend with Django. This figure has shown a consistent upward trend in recent years (see 2021 and 2022 surveys). Additionally, numerous community members and customers have expressed a strong interest in leveraging Django alongside MongoDB.
+Our goal is to meet the needs of the developer community by providing an exceptional experience when working with MongoDB. Therefore, we are embarking on this journey to create a reliable integration. **Please note that the current project is in a very early stage of development and is not yet ready for PoC or Production use cases.**
+
+## Current capabilities
+
+At this stage, the project should let you replicate the djangoproject.com tutorial with minimal changes. You will likely be able to run it passing a portion of the standard Django test suite but not all of it. Some key features that we have added so far are:
```
Comment (timgraham) [inline README.md:113]: This section is too vague to be useful. There are some parts of the list below that will work and some parts that won't. As I have said before, we should document what doesn't work.

File: README.md [hunk 3]
```diff
@@ -149,6 +172,10 @@ from the new application's `apps.py` file.
   - You can study the skipped tests in `DatabaseFeatures.django_test_skips` for
     more details on known issues.
 
+
+## We value your feedback!
+We invite you to share your feedback as we work to address this challenge. If you are interested in becoming a design partner, please connect with us through our community forum. You can also create issues or submit pull requests directly to our project GitHub repository.
```
Comment (timgraham) [inline README.md:177]: What does it mean to be a design partner? Where is the community forum?

File: README.md [hunk 4]
```diff
@@ -101,6 +101,29 @@ from the new application's `apps.py` file.
    Valid values for `verbosity` are `"queryPlanner"` (default),
    `"executionStats"`, and `"allPlansExecution"`.
 
+## Why are we building a MongoDB-Django integration?
```
Comment (timgraham) [inline README.md:104]: Putting this background information amid technical details is out of place. I wouldn't personally include this information in this document.

**Review by timgraham — COMMENTED** [review]
I think the spirit of these changes could be done more succinctly primarily by expanding the first paragraph of this document. As it stands, the additions feel redundant (for example, stating "not for production use" three times) and poorly structured (compared to what I've done in similar projects). I'll propose an alternative.

File: README.md (hunk 4 above)
Comment (Jibola) [reply README.md:104]: What about moving it to the top of the document?

File: README.md (hunk 1 above)
Comment (Jibola) [reply README.md:123]: Yep. Thanks for catching that.

**Comment by timgraham** [comment]
Superseded by https://github.com/mongodb-labs/django-mongodb/pull/138.

**Pull request closed without merging**